private/bufpkg/bufimage/bufimagemodify/internal/bufimagemodifytesting/bufimagemodifytesting.go

```diff
@@ -71,8 +131,31 @@ func AssertFileOptionSourceCodeInfoNotEmpty(t *testing.T, image bufimage.Image,
 	}
 }
 
-func GetTestImage(t *testing.T, dirPath string, includeSourceInfo bool) bufimage.Image {
-	module := GetTestModule(t, dirPath)
+// AssertFileOptionSourceCodeInfoNotEmptyForFile asserts that
+// the source code info for the file option path is present for the file.
+func AssertFileOptionSourceCodeInfoNotEmptyForFile(
+	t *testing.T,
+	imageFile bufimage.ImageFile,
+	fileOptionPath []int32,
+) {
+	descriptor := imageFile.Proto()
+	var hasFileOption bool
+	for _, location := range descriptor.SourceCodeInfo.Location {
+		if len(location.Path) > 0 && internal.Int32SliceIsEqual(location.Path, fileOptionPath) {
+			hasFileOption = true
+			break
+		}
+	}
+	assert.True(t, hasFileOption)
+}
+
+// GetTestImage returns an image from a directory.
```

Comment:
needs justification as to why we need this, and why its not part of a common testing package for images (which likely means we need a TODO to move such a function into a common `bufimagetesting`).

Reply:
Updated the doc explaining it allows skipping creating a module, and also added a TODO to move this to `bufimagetesting`.